docker-compose.yml

```diff
@@ -1,7 +1,7 @@
-version: "3.8"
+version: "3.7"
```

Comment:
Presently, in the Focal env, `version: "3.8"` is unsupported; see https://github.com/travis-ci/docs-travis-ci-com/pull/3025. Downgrading this makes it so that if anyone does move to run tests Dockerized in Travis CI again, they at least won't hit the snag I did.